src/VisualStudio/Core/Test/Venus/DocumentService_IntegrationTests.vb

```diff
@@ -180,7 +182,19 @@ class C { }
                 Assert.False(document.CanApplyChange())
 
                 ' see whether changes can be applied to the solution
-                Assert.Throws(Of NotSupportedException)(Sub() workspace.TryApplyChanges(newDocument.Project.Solution))
+                If ignoreUnchangeableDocuments Then
+                    Assert.True(workspace.IgnoreUnchangeableDocumentsWhenApplyingChanges)
```

Comment:
If you do Assert.Equal() you can lift this out of the If block since you're testing both branches.